.github/workflows/namd.yml

```diff
@@ -3,7 +3,7 @@ name: NAMD
 on: pull_request
 
 jobs:
-  build:
+  build-and-test-namd:
```

Comment:
Maybe add a comment here saying that the name makes it optional?

Reply:
Good idea, added.